fuel-tx/src/builder.rs

```diff
@@ -129,11 +116,10 @@ pub struct TransactionBuilder<Tx> {
 impl TransactionBuilder<Script> {
     pub fn script(script: Vec<u8>, script_data: Vec<u8>) -> Self {
         let tx = Script {
-            gas_price: Default::default(),
             gas_limit: Default::default(),
-            maturity: Default::default(),
             script,
             script_data,
+            policies: Policies::new().with_gas_price(0),
```

Comment:
Because of this check:
<img width="636" alt="image" src="https://github.com/FuelLabs/fuel-vm/assets/18346821/f5ad7785-56ac-45a6-81d9-f5f9161f0b15">

We use builders mostly for tests and it is better to set this policy for all tests instead of updating all places.